http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -248,7 +248,8 @@ class ClusterMonitorActor(
       now <- IO(Instant.now).unsafeToFuture()
       _ <- dbRef.inTransaction { _.clusterQuery.setToRunning(cluster.id, publicIp, now) }
       // Record metrics in NewRelic
-      _ <- recordMetrics(cluster.status, ClusterStatus.Running).unsafeToFuture()
+      _ <- recordStatusTransitionMetrics(cluster.status, ClusterStatus.Running).unsafeToFuture()
+      _ <- recordClusterCreationMetrics(cluster.auditInfo.createdDate, cluster.clusterImages).unsafeToFuture()
```

Comment:
I think this might also be cluster transitioning from `Starting -> Running`, so you'll need to differentiate it from `Creating -> Starting`

Reply:
Good catch, thank you!